example/redis/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM redis:latest
```

Comment:
I might be missing something obvious here, but how does the index.js actually get used in the image? Wouldn't the Dockerfile need to have the nodejs runtime and wire up the index.js as an ENTRYPOINT/CMD?

Reply:
in this case, the docker file is there just to spin up the service the example is connecting to

Reply:
Ah, that makes sense, thanks